src/__tests__/winston_transport.test.ts

```diff
@@ -41,13 +41,13 @@ describe("BaseLogger", () => {
   })
 
   it("carries arguments passed to child loggers", () => {
-    const childLogger = logger.child({ child: "foo" })
+    const childLogger = logger.child({ child: "foo", group: "childgroup" })
     childLogger.info("child logger message", { argument: 123 })
     expect(client.extension.log).toHaveBeenCalledWith(
-      "groupname",
+      "childgroup",
       3,
       "child logger message",
-      { child: "foo", argument: 123 }
+      { child: "foo", argument: 123, group: "childgroup" }
```

Comment:
I think ideally the group wouldn't _also_ appear as an attribute, if it's being used as the group. The implementation should remove the group from the attributes map.